qipr_approver/approver/views/userlist.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth.decorators import user_passes_test
+from django.contrib.auth.models import User
+from django.shortcuts import render
+import approver.utils as utils
+from django.contrib.auth.decorators import user_passes_test
+from approver.decorators import login_required
+from django.http import HttpResponseRedirect
+from approver.constants import users_per_page
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+#@login_required
+#@user_passes_test(lambda u: u.is_superuser)
+def userlist(request):
+	if utils.get_user_from_http_request(request).is_superuser:
+		if request.method == 'GET':
+			users = []
+			userlist = User.objects.all()
```

Comment:
There is a problem here. We have django admin users which are not "people" in the system. When the html tries to render a link to the about you page with a non-person, it breaks.  We should only see a list of people registered through the system, not users. @PFWhite does this sound right?